test/cli.js

```diff
@@ -368,3 +371,14 @@ test('prefers local version of ava', function (t) {
 	t.ok(debugSpy.calledWith('Using local install of AVA'));
 	t.end();
 });
+
+test('use current working directory if `package.json` is not found', function () {
+	var cwd = uniqueTempDir({create: true});
+	var testFilePath = path.join(cwd, 'test.js');
+	var cliPath = require.resolve('../cli.js');
+	var avaPath = require.resolve('../');
+
+	fs.writeFileSync(testFilePath, 'import test from \'' + avaPath + '\';\ntest(t => { t.pass(); });');
```

Comment:
Changing to this probably fixes the AppVeyor error:

```js
'import test from ' + JSON.stringify(avaPath) + ';\ntest(t => { t.pass(); });'
```

I reckon on Windows the path contains backslashes which are interpreted as escape characters.